src/main/java/hudson/plugins/git/opt/PreBuildMergeOptions.java

```diff
@@ -22,17 +22,27 @@
     public RemoteConfig mergeRemote = null;
 
     /**
-     * Remote ref to merge.
+     * Remote ref to merge to.
      */
     public String mergeTarget = null;
-
+    /**
+     * Remote ref to merge from.
+     */
+    public String mergeSource = null;
```

Comment:
No new public fields, please. This is a really bad practice especially if there is a concurrent usage